packages/fast-components-styles-msft/src/select/index.ts

```diff
@@ -13,44 +15,53 @@ import {
 } from "../utilities/color";
 import { applyFloatingCornerRadius } from "../utilities/border";
 import { designUnit } from "../utilities/design-system";
-
-export const selectDisplayButtonOverrides: ComponentStyles<
-    Partial<ButtonClassNameContract>,
-    DesignSystem
-> = {
-    button: {
-        width: "100%",
-        padding: "0 10px",
-    },
-    button_contentRegion: {
-        width: "100%",
-        display: "inline-flex",
-    },
-};
+import { inputFieldStyles } from "../patterns/input-field";
+import { applyDisabledState } from "../utilities/disabled";
+import { focusOutlineWidth } from "../utilities/design-system";
 
 const styles: ComponentStyles<SelectClassNameContract, DesignSystem> = (
     config: DesignSystem
```

Comment:
Since you don't need the designSystem anymore, you can clean up this nested return. Essentially get rid of the ComponentStyleSheet and the "return {". @nicholasrice will be happy :)

Reply:
leaving for now per our discussion

Reply:
(ie - we discovered we still need designsystem to get direction at this time)

Reply:
I'm re-opening this as the discussion was not had on an open thread, if the reasoning is that we need any part of the design system this is possible without making this a function, an example can be found here: https://github.com/Microsoft/fast-dna/blob/master/packages/fast-components-styles-msft/src/button/index.ts#L206